### PR TITLE
[ixia/Keysight] Adding IxANVL automated deployment using PTF32 topology.

### PR DESCRIPTION
**Pull request by aganguly-keysight**
### Description of PR
Addition of IxANVL PTF Container automated deployment using PTF32 topology.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### How did you do it?
Modified ansible/roles/vm_set/tasks/add-topo.yml, ansible/roles/vm_set/tasks/remove-topo.yml, ansible/roles/vm_set/tasks/main.yml to create and bind IxANVL container in a PTF32 topology with SONIC-VS DUT.
Added a new configuration - "ixanvl-vs-conf" in testbed.csv and testbed.yaml for this.

Found one issue during PTF32 topology deployment as the bind was throwing errors as there was no VMs needed for this topology and hence was not defined in topology ymls. Fixed that error by making "vm_names" to be an optional argument in 
ansible/roles/vm_set/library/vm_topology.py main function. Made subsequent changes in that file for this purpose.

#### How did you verify/test it?
By running testbed-cli.sh script for add-topo and remove-topo for ixanvl-vs-conf configuration.
### Documentation 
A new README added in testbed section.
Readme file: README.testbed.IxANVL.VsSetup.md

### CODE REVIEW

File: ansible/roles/vm_set/library/vm_topology.py [hunk 1]
```diff
@@ -923,7 +922,7 @@ def main():
             cmd=dict(required=True, choices=['create', 'bind', 'renumber', 'unbind', 'destroy', "connect-vms", "disconnect-vms"]),
             vm_set_name=dict(required=False, type='str'),
             topo=dict(required=False, type='dict'),
-            vm_names=dict(required=True, type='list'),
+            vm_names=dict(required=False, type='list'),
```
Comment (yxieca) [inline ansible/roles/vm_set/library/vm_topology.py:925]: I think @neethajohn asked for a similar change in another PR. please don't allow skip this option but allow provide empty list.

File: ansible/roles/vm_set/library/vm_topology.py [hunk 2]
```diff
@@ -941,10 +940,13 @@ def main():
...
 
     cmd = module.params['cmd']
-    vm_names = module.params['vm_names']
     fp_mtu = module.params['fp_mtu']
     max_fp_num = module.params['max_fp_num']
     duts_mgmt_port = []
+    if 'vm_names' not in module.params:
+        vm_names = []
+    else:
+        vm_names = module.params['vm_names']
```
Comment (yxieca) [inline ansible/roles/vm_set/library/vm_topology.py:949]: As above.

Comment (aganguly-keysight) [reply ansible/roles/vm_set/library/vm_topology.py]: Hi @yxieca 

I have updated vm_topology.py as per the review comments. 
Please share your feedback on the latest changes.

**Review by yxieca — APPROVED** [review]

**Comment by aganguly-keysight** [comment]
Hello @yxieca 
I have updated the PR with changes for running bgp conformance tests. Updated conftest.py and added test_bgp_conformance.py . Please share your review comments.

**Pull request merged**